README.md

```diff
@@ -55,3 +55,23 @@ key and all necessary information are provided.
 ## Usage
 
 Run `astartectl` to see available commands.
+
+### Send or set data to devices
+
+For server-owned interfaces, it is possible to send or set data via astartectl.
+To do so, you can use:
+- astartectl appengine devices send-data (deprecated)
+- astartectl appengine devices publish-datastream
+- astartectl appengine devices set-property
+
+Run the appropriate command to see a short description, command suggestions and flags.
+Generally speaking, one of those could be written like 
+```
+ astartectl appengine devices publish-datastream 2TBn-jNESuuHamE2Zo1anA com.my.interface /my/path "value"
```

Comment:
This is simple enough for individual data.
A similar description for aggregates would help.

Reply:
For example you could show that for an aggregated object the value should be in the json form:
`"{"path1": value2,"path2":value2"}"`